src/devices/shelly.ts

```diff
@@ -13,21 +13,27 @@ export const definitions: DefinitionWithExtend[] = [
         fingerprint: [{modelID: "1", manufacturerName: "Shelly"}],
         model: "S4SW-001X16EU",
         vendor: "Shelly",
-        description: "1 gen 4",
+        description: "1 Gen 4",
         extend: [m.onOff({powerOnBehavior: false})],
     },
     {
         zigbeeModel: ["Mini1PM"],
         model: "S4SW-001P8EU",
         vendor: "Shelly",
         description: "1PM Mini Gen 4",
-        extend: [m.onOff({powerOnBehavior: false}), m.electricityMeter({energy: false, producedEnergy: true, acFrequency: true})],
+        extend: [
+            m.onOff({powerOnBehavior: false}),
+            m.electricityMeter({producedEnergy: true, acFrequency: true, energy: {change: 1}, current: {change: 2}, power: {change: 100}}),
+        ],
     },
     {
         zigbeeModel: ["1PM"],
         model: "S4SW-001P16EU",
         vendor: "Shelly",
         description: "1PM Gen 4",
-        extend: [m.onOff({powerOnBehavior: false}), m.electricityMeter({energy: false, producedEnergy: true, acFrequency: true})],
+        extend: [
+            m.onOff({powerOnBehavior: false}),
+            m.electricityMeter({producedEnergy: true, acFrequency: true, energy: {change: 1}, current: {change: 2}, power: {change: 100}}),
```

Comment:
```suggestion
            m.electricityMeter({producedEnergy: true, acFrequency: true, energy: {change: 1}, current: {change: 2}, power: {change: 100}}),
```

Can you try without the `change` for power/current and energy?

Reply:
Yeah this should work too